genesyscloud/oauth_client/resource_genesyscloud_oauth_client_schema.go

```diff
@@ -73,10 +74,10 @@ func ResourceOAuthClient() *schema.Resource {
 				Optional:    true,
 			},
 			"authorized_grant_type": {
-				Description:  "The OAuth Grant/Client type supported by this client (CODE | TOKEN | SAML2BEARER | PASSWORD | CLIENT-CREDENTIALS).",
+				Description:  "The OAuth Grant/Client type supported by this client (CODE | TOKEN | SAML2-BEARER | PASSWORD | CLIENT-CREDENTIALS).",
```

Comment:
`SAML2-BEARER` is what is output via the API, which is why I chose to use this in the description examples instead of `SAML2BEARER`. In reality, both values are perfectly acceptable.